frontend/components/UserSettings.tsx

```diff
@@ -86,47 +87,70 @@ const majorOptions = [
   { code: 'SPA', name: 'Spanish' },
   { code: 'POR', name: 'Portuguese' },
   { code: 'SPI', name: 'School of Public and International Affairs' },
-  // { code: 'Independent', name: 'Independent' },
+  // { code: 'Independent', name: 'Independent' }, TODO: This is an actual major here and should be added.
   { code: 'Undeclared', name: 'Undeclared' },
 ];
 
 const minorOptions = [
+  { code: 'AAS', name: 'African American Studies' },
   { code: 'AFS', name: 'African Studies' },
+  { code: 'APC', name: 'Applied and Computational Mathematics' },
+  { code: 'ARA', name: 'Arabic Language' },
   { code: 'ASA', name: 'Asian American Studies' },
+  { code: 'ART', name: 'Art History' },
+  { code: 'BNG', name: 'Bengali Language and Culture' },
+  { code: 'CGS', name: 'Cognitive Science' },
   { code: 'CHI', name: 'Chinese Language' },
   { code: 'CLA', name: 'Classics' },
   { code: 'COS', name: 'Computer Science' },
   { code: 'CS', name: 'Climate Science' },
   { code: 'CWR', name: 'Creative Writing' },
   { code: 'DAN', name: 'Dance' },
-  { code: 'EAS', name: 'East Asian Studies Program' },
+  { code: 'EAS', name: 'East Asian Studies' },
+  { code: 'ENE', name: 'Sustainable Energy' },
   { code: 'ENG', name: 'English' },
   { code: 'ENV', name: 'Environmental Studies' },
+  { code: 'EUS', name: 'European Studies' },
   { code: 'FIN', name: 'Finance' },
+  { code: 'FRE', name: 'French Language and Culture' },
   { code: 'GHP', name: 'Global Health and Health Policy' },
   { code: 'GSS', name: 'Gender and Sexuality Studies' },
+  { code: 'HOA', name: 'History of Art' },
```

Comment:
I was going to make a commit in HART.yaml in its branch, but since "History of Science" is HOS, I think "History of Art" should also be HOA. Plus, it's 3 letters which is more fitting as a Princeton abbrev. Thoughts?

Reply:
Yea that sounds good

Reply:
There might be a merge conflict if this has HOA and the other has HART in the YAML file. Since I only changed UserSettings.tsx, we can just change HOA in UserSettings.tsx in this commit and then modify the YAML file contents in `issac-history-of-art-addition` branch to reflect this.

Reply:
Cool let's do that then